docs/en/sql-reference/functions/other-functions.md

```diff
@@ -221,6 +221,38 @@ This function returns the size (row count) of the block the function is called o
 blockSize()
 ```
 
+**Example**
+
+Query:
```

Comment:
The example is a bit confusing, let's do this:

```
DROP TABLE IF EXISTS test;
CREATE TABLE test (n UInt8) ENGINE = Memory;

INSERT INTO test
    SELECT * FROM system.numbers LIMIT 5;

SELECT blockSize()
FROM test
SETTINGS max_block_size = 2;
```

Please first see my explanation about data parts vs blocks in my other comment.

The new example should yield this result:

```
2
2
--
2
2
-- 
1
```

It doesn't, perhaps there is a bug or something I don't understand. So let's dumb it further down:

```
DROP TABLE IF EXISTS test;
CREATE TABLE test (n UInt8) ENGINE = Memory;

INSERT INTO test
    SELECT * FROM system.numbers LIMIT 5;

SELECT blockSize()
FROM test;
```

Returns

```
5
5
5
5
5
```